pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -103,13 +103,20 @@ func (q *QueryReconciler) collectInfoFromVdb(ctx context.Context) (ctrl.Result,
 	return res, err
 }
 
-// setListRestorePointsQueryConditions will update the status condition before and after calling
+// runListRestorePoints will update the status condition before and after calling
 // list restore points api
 // Temporarily, runListRestorePoints will not call the ListRestorePoints API
 // since the dispatcher is not set up yet
 func (q *QueryReconciler) runListRestorePoints(ctx context.Context, _ *ctrl.Request) error {
+	// update query message prior to calling vclusterops API
+	err := vrpqstatus.UpdateMessageStatus(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
+		"Querying")
+	if err != nil {
+		return err
+	}
+
 	// set Querying status condition prior to calling vclusterops API
-	err := vrpqstatus.UpdateCondition(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
+	err = vrpqstatus.UpdateCondition(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
 		v1.MakeCondition(vapi.Querying, metav1.ConditionTrue, ""))
 	if err != nil {
 		return err
```

Comment:
Since we have no error codepath yet, can we add a comment (and jira) to line 125 (`// API should be called to proceed here`) so we don't forget.